editor/editor_help.cpp

```diff
@@ -721,18 +723,43 @@ void EditorHelp::_update_doc() {
 				property_descr = true;
 			}
 
+			class_desc->pop();
+			class_desc->pop();
+			class_desc->pop(); // cell
+
+			// Property value.
+			class_desc->push_cell();
+			class_desc->push_font(doc_code_font);
+
 			if (cd.properties[i].default_value != "") {
 				class_desc->push_color(symbol_color);
-				class_desc->add_text(cd.properties[i].overridden ? " [" + TTR("override:") + " " : " [" + TTR("default:") + " ");
+				if (cd.properties[i].overridden) {
+					class_desc->add_text(" [" + TTR("overrides") + " ");
+					class_desc->push_meta("@member " + cd.properties[i].overrides + "." + cd.properties[i].name);
+					_add_text(cd.properties[i].overrides);
+					class_desc->pop();
+					class_desc->add_text(": ");
```

Comment:
That's a bit problematic for translation if some languages need to change the order of words in the sentence, or the punctuation. E.g. French would need to have a space before `:`, and I assume some languages might need to put the class name first before the verb?

But it's tricky to do any other way due to the `push_meta`.

Reply:
I guess we can make the whole line the text for the meta link? And make the translation string with a placeholder.

Reply:
![image](https://user-images.githubusercontent.com/11782833/144621227-98291cbd-e2a2-463b-91d2-fa2f0d3eef95.png)

Reply:
Well, I've pushed it like that :)